### PR TITLE
Plot events

### PR DESCRIPTION
**Pull request by Bobfrat**
@DanielJMaher @FBRTMaka @birdage 

Plots events, and also fixes bug in interpolated plot.

Goes with https://github.com/asascience-open/ooi-ui-services/pull/434

### CODE REVIEW

File: ooiui/static/js/views/science/PlotAnnotationsEventsButtonView.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/*
+ * ooiui/static/js/views/common/ModalDialogView.js
```
Comment (birdage) [inline ooiui/static/js/views/science/PlotAnnotationsEventsButtonView.js]: can we rename this plssssss

**Comment by birdage** [comment]
:+1:

File: ooiui/static/js/views/science/PlotAnnotationsEventsButtonView.js (hunk 1 above)
Comment (Bobfrat) [inline ooiui/static/js/views/science/PlotAnnotationsEventsButtonView.js]: I was just about to then it got merged!

Comment (FBRTMaka) [inline ooiui/static/js/views/science/PlotAnnotationsEventsButtonView.js]: Ha! Now we can all have fun with PlotAnnotationsEventsButtonViewNameThatIsSoLongDotJS


On Nov 6, 2015, at 1:43 PM, Bob Fratantonio <notifications@github.com<mailto:notifications@github.com>> wrote:


In ooiui/static/js/views/science/PlotAnnotationsEventsButtonView.js<https://github.com/asascience-open/ooi-ui/pull/579#discussion_r44172183>:

> @@ -0,0 +1,32 @@
> +/*
> + * ooiui/static/js/views/common/ModalDialogView.js


I was just about to then it got merged!

—
Reply to this email directly or view it on GitHub<https://github.com/asascience-open/ooi-ui/pull/579/files#r44172183>.


This e-mail message and any attached file is the property of the sender and is sent in confidence to the addressee only.
Internet communications are not secure and RPS is not responsible for their abuse by third parties, any alteration or corruption in transmission or for any loss or damage caused by a virus or by other means.
 
Any advice contained in this e-mail is for information purposes only.

RPS Group Plc, company number: 208 7786 (England). Registered office: 20 Western Avenue Milton Park Abingdon Oxfordshire OX14 4SH.
RPS Group Plc web link: <http://www.rpsgroup.com>

**Pull request merged**